ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/confidential/confidentialvm/provision/cucushift-installer-rehearse-azure-ipi-confidential-confidentialvm-provision-chain.yaml

```diff
@@ -0,0 +1,22 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-confidential-confidentialvm-provision
+  steps:
+  - chain: ipi-conf-azure
+  - ref: ipi-conf-azure-confidential-confidentialvm
+  - ref: ipi-conf-azure-custom-region
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
+  - chain: cucushift-installer-check
+  - ref: cucushift-installer-check-azure-confidential-confidentialvm
+  env:
+  - name: CUSTOM_AZURE_REGION
+    default: "eastus"
+    documentation: |-
+      Specify the region where vm size supports ConfidentailVM
+  - name: OS_DISK_SECURITY_ENCRYPTION_TYPE
+    default: "VMGuestStateOnly"
+    documentation: |-
+      Specify the encryption type of the managed disk
+      Valid Values: "VMGuestStateOnly","DiskWithVMGuestState"
```

Comment:
Do we have plan to cover `DiskWithVMGuestState` ?

Reply:
`DiskWithVMGuestState` is blocked by https://issues.redhat.com/browse/OCPBUGS-18379 ([RHEL-16267](https://issues.redhat.com/browse/RHEL-16267)), will add related job later once it is supported.